clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -182,6 +182,17 @@ public class CommonClientConfigs {
     public static final String DEFAULT_API_TIMEOUT_MS_DOC = "Specifies the timeout (in milliseconds) for client APIs. " +
             "This configuration is used as the default timeout for all client operations that do not specify a <code>timeout</code> parameter.";
 
+    /**
+     * <code>internal.leave.group.on.close</code>
+     * Whether or not the consumer should leave the group on close. If set to <code>false</code> then a rebalance
+     * won't occur until <code>session.timeout.ms</code> expires.
+     *
+     * <p>
+     * Note: this is an internal configuration and could be changed in the future in a backward incompatible way
+     *
+     */
+    public static final String LEAVE_GROUP_ON_CLOSE_CONFIG = "internal.leave.group.on.close";
```

Comment:
Any reason why we move the config from `ConsumerConfigs` to `CommonClientConfigs`? This is a consumer config, isn't it?

Reply:
Oh, you are correct, we just need to change `ConsumerConfig.LEAVE_GROUP_ON_CLOSE_CONFIG` to `public` to allow `StreamsConfig` to access this configuration.
Thanks.

Reply:
Hi @showuon Sorry, I remembered why it was changed in the first place.
Because `GroupRebalanceConfig` does not allow us to import `ConsumerConfig`, `ConsumerConfig.LEAVE_GROUP_ON_CLOSE_CONFIG` cannot be used. So I moved `LEAVE_GROUP_ON_CLOSE_CONFIG` to `CommonClientConfigs`.

Reply:
OK, I see. Then we can just leave the `GroupRebalanceConfig` as is, and change other places. The main reason is the config `LEAVE_GROUP_ON_CLOSE_CONFIG` is only used in Consumer, not other places, so we should keep it there. Thanks.

Reply:
Sure